tdesign/src/Layout/BaseLayout/BaseLayout.tsx

```diff
@@ -34,6 +35,7 @@ const BaseLayout = ({
   logoText,
   brandName,
   logoLinkTo = "/",
+  marketingNotice = null,
```

Comment:
Ah, well that explains why we have so many errors, because I use this pattern all over the place lol.

I thought that if `marketingNotice` is `undefined` then you cannot do `{marketingNotice}` in JSX, which is why I default it to `null`, but I just checked and was surprised to see that's not the case.